docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -408,7 +414,7 @@ curl \
   -d '[{ "op": "remove", "path": "/status/bootstrapToken" }]' \
   "https://rancher.vshn.net/k8s/clusters/c-c6j2w/apis/syn.tools/v1alpha1/namespaces/${LIEUTENANT_NS}/clusters/${CLUSTER_ID}/status"
 
-kubectl apply -f $(http -pb "https://${LIEUTENANT_URL}/clusters/${CLUSTER_ID}" "${LIEUTENANT_AUTH}" | jq -r ".installURL")
+kubectl apply -f $(curl -sH "${LIEUTENANT_AUTH}" "https://${LIEUTENANT_URL}/clusters/${CLUSTER_ID}" | jq -r ".installURL")
```

Comment:
Can we set the `--kubeconfig` here? It happened to me several times, that the wrong Kubeconfig was used.

Reply:
It should be set by the `KUBECONFIG` env var of the previous step. But yes, setting it explicitly might be a better solution.

Reply:
I have added `--kubeconfig "$(pwd)/../../../target/auth/kubeconfig"`, that right?

Reply:
Depends on the directory you're currently in 🤷‍♂️